src/new-client/jest.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
+  testRegex: "\.test.[t]sx?$",
```

Comment:
I'm pretty sure that the default `testRegex` already matches this, right? If so, I would rather to stick with the default value.

Reply:
> This narrows actually narrows the match to force the convention we're using

I rather use the standard convention than to create our own. Sorry, but I couldn't resist using this meme :see_no_evil: 
![image](https://user-images.githubusercontent.com/8620144/111506726-35826600-874a-11eb-8719-3f43bdabb945.png)

> it also lets us include files in a tests folder that are not tests, like shared mock data, without manually exempting them from the runner

I would prefer not to have files on the tests folder that are not tests. Mock data files belong into their respective `__mocks__` folders. For instance, if we exposed the mocked data that's on the `src/App/Trades/__tests__/mockTrades.ts` through the `src/services/trades/__mocks__/` then we will be able to re-use those mocks for other tests that are mocking the `services/trades`